src/netstandard/Microsoft.ServiceFabric.AspNetCore.Configuration/ServiceFabricConfigurationExtensions.cs

```diff
@@ -0,0 +1,61 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.ServiceFabric.AspNetCore.Configuration
```

Comment:
>namespace [](start = 0, length = 9)

This file is not needed in netstandard folder, netstandard .csproj will use the file from full framework folder.

Reply:
fixed.